live-flowchart/sw.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * This is a service worker
+ */
+
+console.debug('\nsw.js: this is a service worker');
```

Comment:
`\n` is to write a new line. In sw.js to separate each state of the SW execution (`init`, `oninstall`, `onactivate`, `onfetch`), to make it easier to read. I can do the same with a `console.debug('\n')` or a `console.debug('---')` before instead.